modules/manifold/src/components/manifold.js

```diff
@@ -174,161 +181,156 @@ class Manifold extends PureComponent {
       return <ErrorContainer> {dataLoadingError.message} </ErrorContainer>;
     }
     return (
-      <StyletronProvider value={engine}>
-        <BaseProvider theme={THEME}>
-          <Container width={width} height={height}>
-            <Content viewMode={viewMode}>
-              <Panel
-                key="headline"
-                gridArea="headline"
-                isShown={!showBoth}
-                borderBottom
-                padded
-                flexDirection="row"
-              >
-                <Headline
-                  headers={[VIEW_NAME.PERF, VIEW_NAME.FEATURE]}
-                  onTabChange={this._toggleViewTab}
-                  isCoordinated={showBoth}
-                  showHelp={helpType === HELP_TYPE.PERF}
-                  onClickSplit={this._toggleViewMode}
-                  onClickHelp={() =>
-                    this._toggleHelpType(
-                      showLeftView ? HELP_TYPE.PERF : HELP_TYPE.FEATURE
-                    )
-                  }
-                  themeColor={THEME_COLOR}
-                />
-              </Panel>
-              <Panel
-                key="headline1"
-                gridArea="headline1"
-                isShown={showBoth}
-                borderBottom
-                borderRight
-                padded
-                flexDirection="row"
-              >
-                <Headline
-                  headers={[VIEW_NAME.PERF]}
-                  themeColor={THEME_COLOR}
-                  isCoordinated={showBoth}
-                  showHelp={helpType === HELP_TYPE.PERF}
-                  onClickSplit={this._toggleViewMode}
-                  onClickHelp={() => this._toggleHelpType(HELP_TYPE.PERF)}
-                />
-              </Panel>
-              <Panel
-                key="headline2"
-                gridArea="headline2"
-                isShown={showBoth}
-                borderBottom
-                flexDirection="row"
-              >
-                <Headline
-                  headers={[VIEW_NAME.FEATURE]}
-                  themeColor={THEME_COLOR}
-                  isCoordinated={showBoth}
-                  showHelp={helpType === HELP_TYPE.FEATURE}
-                  onClickSplit={this._toggleViewMode}
-                  onClickHelp={() => this._toggleHelpType(HELP_TYPE.FEATURE)}
-                />
-              </Panel>
+      // <StyletronProvider value={engine}>
+      <BaseProvider theme={theme || THEME}>
```

Comment:
Do we need to have this if the container app has one already?

Reply:
We need to have some them provider so as to ensure the default `THEME` is provided. Although there might be some other provider other than `BaseProvider` that we can use. Is that what you mean?

Reply:
I think if the `StyletronProvider` exists in the parent app, then the `BaseProvider` should be there as well. Not sure if we could just use the parent's BaseProvider and theme instead of having to pass the theme along to manifold

Reply:
I see. But manifold defaults to using some theme overrides, and we need to have a provider for that.